src/game/engine/house.cpp

```diff
@@ -823,6 +824,93 @@ BOOL HouseClass::Is_Ally(ObjectClass *object) const
     return Is_Ally(object->Owner());
 }
 
+ProdFailType HouseClass::Begin_Production(RTTIType rtti, int id)
```

Comment:
not sure on the position in the header and cpp for these functions, any specific order they should follow?